### PR TITLE
Update to dojot version 0.2.0

### PR DESCRIPTION
**Pull request by ericbaum**
This pull requests updates the manifests to dojot version 0.2.0

- Adds a Kafka cluster
- Adds the Data Broker module
- Adds RabbitMQ
- Changes STH for the History and Persister modules
- Changes the iotagent-json and the mqtt agent for the iotagent-mosca
- Replaces mashup and perseo with the flowbroker

- Adds an initial configuration file for the environment containing:
  - External access configuration
  - Version configuration
  - Storage configuration
  - Cluster Size configuration for clusterizable services.

Resolves dojot/dojot#287

### CODE REVIEW

File: config_scripts/kong.config.sh [hunk 1]
```diff
@@ -142,47 +162,45 @@ authConfig "flows"
...
-    "uris": "/mashup",
+    "name": "flows",
+    "uris": ["/flows"],
     "strip_uri": true,
-    "upstream_url": "http://mashup:1880"
+    "upstream_url": "http://flowbroker:80"
 }
 PAYLOAD
-# no auth: serves only available types
+authConfig "flows"
```
Comment (mmagr) [inline config_scripts/kong.config.sh]: This has to be updated, no?

Comment (ericbaum) [reply config_scripts/kong.config.sh]: yep, will do

Comment (ericbaum) [reply config_scripts/kong.config.sh]: Looking at it again, no, it doesn't require any changes, there are issues only when the auth is turned on for the /mashup endpoint, which is not happening on this PR. 

This PR only set the auth for the "/flows" endpoint and it has been working correctly so far.

Comment (mmagr) [reply config_scripts/kong.config.sh]: True enough. My bad.

**Pull request merged**